xbmc/windowing/win10/WinSystemWin10DX.cpp

```diff
@@ -61,6 +61,10 @@ bool CWinSystemWin10DX::CreateNewWindow(const std::string& name, bool fullScreen
   m_deviceResources = DX::DeviceResources::Get();
   m_deviceResources->SetWindow(m_coreWindow);
 
+  m_uiThreadId = GetCurrentThreadId();
+  IsHDRDisplay();
+  HasSystemSdrPeakLuminance();
```

Comment:
I trust that you runtime tested that those calls are valid before the window creation.
Please add a comment that the calls exist to initialize the cached values. It's weird to throw away the returned values otherwise.

Reply:
Done